lib/response.js

```diff
@@ -25,7 +25,7 @@ var sign = require('cookie-signature').sign;
 var normalizeType = require('./utils').normalizeType;
 var normalizeTypes = require('./utils').normalizeTypes;
 var setCharset = require('./utils').setCharset;
-var statusCodes = http.STATUS_CODES;
+var statusCodes = require('statuses/codes')
```

Comment:
Hm, maybe I am missing something here, but the object found in `statuses/codes` is not directly exposed.  So I don't think I can get `Found` from `302` in the existing api.  I would either need to change the statuses package to expose it, or use the `statuses.codes` array to reverse engineer back to the original data structure.  Of those two, I obviously lean toward updating the statuses package to expose the raw `codes` object.